client/my-sites/plan-features/header.jsx

```diff
@@ -357,7 +386,7 @@ PlanFeaturesHeader.propTypes = {
 	billingTimeFrame: PropTypes.string.isRequired,
 	current: PropTypes.bool,
 	onClick: PropTypes.func,
-	planType: React.PropTypes.oneOf( [ PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS ] ).isRequired,
+	planType: React.PropTypes.string.isRequired,
```

Comment:
Why not one of the constants? I'd like to know because I don't see a difference. :)